README.md

```diff
@@ -21,6 +21,17 @@ NMState is aimed to satisfy enterprise needs to manage host networking through
 a northbound declarative API and multi provider support on the southbound.
 NetworkManager acts as the main (and currently the only) provider supported.
 
+## Contact
+*nmstate* uses the Network [Manager mailing
+list](https://mail.gnome.org/mailman/listinfo/networkmanager-list)
+([Archives](https://mail.gnome.org/archives/networkmanager-list/)) for
+discussions. Emails about nmstate should be tagged with `[nmstate]` in the
+subject header to ease filtering.
+
+Development planning (sprints and progress reporting) happens in
+([Jira](https://nmstate.atlassian.net)). Access is possible for anyone with an
```

Comment:
You could say 'Access requires login', mentioning account sounds like it is something special. Not a must.

Reply:
addressed and also added IRC